File: Dataset/Concurrency_prepared/Java/concurrent-computing-1_trace_4.java
Thread[] threads = new Thread[3];
threads[0] = new Thread(() -> System.out.println("enjoy"));
threads[1] = new Thread(() -> System.out.println("rosetta"));
threads[2] = new Thread(() -> System.out.println("code"));
Collections.shuffle(Arrays.asList(threads));
for (Thread thread : threads)
    thread.start();

public class TestThreadShuffle {
    public static void main(String[] args) {
        Thread[] threads = new Thread[3];
        threads[0] = new Thread(() -> System.out.println("enjoy"));
        threads[1] = new Thread(() -> System.out.println("rosetta"));
        threads[2] = new Thread(() -> System.out.println("code"));
        Collections.shuffle(Arrays.asList(threads));
        for (Thread thread : threads)
            thread.start();
    }
}

File: Dataset/Concurrency_prepared/Python/concurrent-computing-1_trace_4.py
async def print_(string: str) -> None:
    print(string)

async def main():
    strings = ['Enjoy', 'Rosetta','Code','Whole','Lot']
    coroutines = map(print_, strings)
    await asyncio.gather(*coroutines) #START


def run_async_code():
    loop = asyncio.new_event_loop()
    asyncio.set_event_loop(loop)
    loop.run_until_complete(main())


if __name__ == '__main__':
    run_async_code()
